server/command_create_test.go

```diff
@@ -137,34 +137,6 @@ func TestCreateCommand(t *testing.T) {
 			assert.Contains(t, resp.Text, "Installation being created.")
 		})
 
-		t.Run(cloud.InstallationDatabaseSingleTenantRDSMySQL, func(t *testing.T) {
```

Comment:
Do you think we could update the tests to expect an error instead of removing them?

Reply:
Right. I will do that. Thanks.